src/devices/RealtimeClock/RtcBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Rtc
```

Comment:
Can we rename this to RealtimeClock? It will make it easier for people to find where the namespace is defined on the repo if the namespace matches the folder and project it is located in.

Reply:
I actually think the current namespace makes sense, I'd rather rename the folder to RTC - RTC is really well known abbreviation in hardware so doesn't make sense to expand it

Reply:
rename to Rtc